<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="vL0">
    <item>
     <widget class="QLabel" name="mainTitle">
      <property name="text">
       <string>Covid19 - Evolution</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="wMain" native="true">
      <layout class="QHBoxLayout" name="hL1">
       <item>
        <widget class="QWidget" name="wConfirmed" native="true">
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="vLConfirmed">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="title">
             <string>Total</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="5" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Taux de positivité (7 derniers jours)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLCDNumber" name="lcdPos">
               <property name="digitCount">
                <number>4</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Nombre de tests (total)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLCDNumber" name="lcdTests">
               <property name="digitCount">
                <number>8</number>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLCDNumber" name="lcdIncid">
               <property name="digitCount">
                <number>3</number>
               </property>
               <property name="intValue" stdset="0">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Taux d'incidence (7 derniers jours)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLCDNumber" name="lcdDeaths">
               <property name="font">
                <font>
                 <pointsize>48</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLCDNumber" name="lcdRea">
               <property name="font">
                <font>
                 <pointsize>48</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="digitCount">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Nombre de patients hospitalisés</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLCDNumber" name="lcdHospi"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nombre de morts à l'hopital (total)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Nombre de patients en réa</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Nombre de tests positifs (total)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLCDNumber" name="lcdTestPos">
               <property name="digitCount">
                <number>6</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbAffichage">
            <property name="title">
             <string>Affichage:</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="cbRegions">
               <property name="text">
                <string>Regions</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbDeps">
               <property name="text">
                <string>Départements</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labConfirmed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Administration:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tabData"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wGraph" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="gbGraphOpt">
            <property name="title">
             <string>Choix Affichage</string>
            </property>
            <layout class="QHBoxLayout" name="layRB">
             <item>
              <widget class="QRadioButton" name="rbRea">
               <property name="text">
                <string>Réanimation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbDeaths">
               <property name="text">
                <string>Décès</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbHosp">
               <property name="text">
                <string>Hospitalisation</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTest">
               <property name="text">
                <string>Tx Positivité</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbIncid">
               <property name="text">
                <string>Tx Incidence</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbEvol">
            <property name="title">
             <string>Evolution / Nouveau cas</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="rbEvol">
               <property name="text">
                <string>Evolution</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbNew">
               <property name="text">
                <string>Nouveaux cas</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbSexe">
            <property name="title">
             <string>Sexe:</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="rbH">
               <property name="text">
                <string>Homme</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbF">
               <property name="text">
                <string>Femme</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbTot">
               <property name="text">
                <string>Total</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbAge">
            <property name="title">
             <string>Catégories Ages:</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QRadioButton" name="rbAll">
               <property name="text">
                <string>All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb0">
               <property name="text">
                <string>0-10</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb1">
               <property name="text">
                <string>10-20</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb2">
               <property name="text">
                <string>20-30</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb3">
               <property name="text">
                <string>30-40</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb4">
               <property name="text">
                <string>40-50</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb5">
               <property name="text">
                <string>50-60</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb6">
               <property name="text">
                <string>60-70</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb7">
               <property name="text">
                <string>70-80</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb8">
               <property name="text">
                <string>80-90</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rb9">
               <property name="text">
                <string>90+</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wMpl" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="layMpl"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
